clang/include/clang/Basic/Builtins.td

```diff
@@ -4744,6 +4744,12 @@ def HLSLAny : LangBuiltin<"HLSL_LANG"> {
   let Prototype = "bool(...)";
 }
 
+def HLSLWaveActiveAnyTrue : LangBuiltin<"HLSL_LANG"> {
+  let Spellings = ["__builtin_hlsl_wave_active_any_true"];
+  let Attributes = [NoThrow, Const];
+  let Prototype = "bool(bool)";
```

Comment:
A note: The reason we didn't have to do a SemaHLSL change for `__builtin_hlsl_wave_active_any_true` is because we could define the prototype with only scalars. For HLSL`builtins` that take vectors we define the prototype variadic.